_docs_explore/64-request-items.md

```diff
@@ -0,0 +1,141 @@
+---
```

Comment:
Sollte nicht vielleicht auf dieselbe Seite auch die [RequestItem Combinations Tabelle](https://js-soft.atlassian.net/wiki/spaces/JSSNMSHDD/pages/3327000583/RequestItem+Combinations)? Bzw. man könnte ja die erste Spalte der Tabelle weglassen, und dafür die Tabelle auf die jeweiligen Kapitel splitten.

Reply:
Hätte ich für jedes Item einzeln dazu gemacht. So ne große Tabelle ist in unserer Doku zu unübersichtlich.